jobs/JEVS_ANALYSES_PLOTS

```diff
@@ -57,7 +57,7 @@ export RUN=${RUN:-atmos}
 echo $COMPATH
 
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINanl=${COMINanl:-${COMIN}/stats/${COMPONENT}}
+export EVSINanl=${EVSINanl:-${COMIN}/stats/${COMPONENT}}
```

Comment:
Just a note here ... EVSINanl was not defined in the driver script. But this worked fine b/c COMIN is defined in the driver script and used to define EVSINanl in the J-job.

Reply:
Oh.  So should EVSINanl be defined in the driver?

Reply:
@PerryShafran-NOAA I think this is fine as it is! Defining COMIN in the dev drivers was intended to make everything that uses $COMIN in the J-jobs work!

Reply:
I agree -- I think this is fine. I was pointing it out b/c the PR instructions indicated to change EVSINanl, but I had changed COMIN in the driver instead.